lib/src/interfaces/spi.dart

```diff
@@ -14,26 +14,31 @@ class SpiInterface extends PairInterface {
   /// The data length for serial transmissions on this interface.
   final int dataLength;
 
+  /// The number of Chip Select signals in this interface.
+  final int multiChipSelects;
+
   /// Serial clock (SCLK). Clock signal from main to sub(s).
   Logic get sclk => port('SCLK');
 
   /// Main Out Sub In (MOSI). Serial data from main to sub(s).
   Logic get mosi => port('MOSI');
 
   /// Main In Sub Out (MISO). Serial data from sub(s) to main.
-  Logic get miso => port('MISO');
+  LogicNet get miso => port('MISO') as LogicNet;
 
   /// Chip select (active low). Chip select signal from main to sub.
-  Logic get csb => port('CSB');
+  List<Logic> get csb => List.generate(multiChipSelects, (i) => port('CSB$i'));
 
   /// Creates a new [SpiInterface].
-  SpiInterface({this.dataLength = 1})
+  SpiInterface({this.dataLength = 1, this.multiChipSelects = 1})
```

Comment:
validation: do you want to check that these inputs are valid?  both of these should be `>= 1` right?